raw_map/src/lane_specs.rs

```diff
@@ -1,558 +1,171 @@
-            base.max(1)
-        }
-    } else if oneway {
-        0
-    } else {
-        1
-    };
 
-    #[allow(clippy::if_same_then_else)] // better readability
-    let driving_lane =
-        if tags.is("access", "no") && (tags.is("bus", "yes") || tags.is("psv", "yes")) {
-            // Sup West Seattle
-            LaneType::Bus
-        } else if tags
-            .get("motor_vehicle:conditional")
-            .map(|x| x.starts_with("no"))
-            .unwrap_or(false)
-            && tags.is("bus", "yes")
-        {
-            // Example: 3rd Ave in downtown Seattle
-            LaneType::Bus
-        } else if tags.is("access", "no") || tags.is("highway", "construction") {
-            LaneType::Construction
-        } else {
-            LaneType::Driving
-        };
-
-    // These are ordered from the road center, going outwards. Most of the members of fwd_side will
-    // have Direction::Fwd, but there can be exceptions with two-way cycletracks.
-    let mut fwd_side: Vec<LaneSpec> = iter::repeat_with(|| fwd(driving_lane))
-        .take(num_driving_fwd)
-        .collect();
-    let mut back_side: Vec<LaneSpec> = iter::repeat_with(|| back(driving_lane))
-        .take(num_driving_back)
-        .collect();
-    // TODO Fix upstream. https://wiki.openstreetmap.org/wiki/Key:centre_turn_lane
-    if tags.is("lanes:both_ways", "1") || tags.is("centre_turn_lane", "yes") {
-        fwd_side.insert(0, fwd(LaneType::SharedLeftTurn));
-    }
-
-    if driving_lane == LaneType::Construction {
-        return LaneSpec::assemble_ltr(fwd_side, back_side, cfg.driving_side);
-    }
-
-    let fwd_bus_spec = if let Some(s) = tags.get("bus:lanes:forward") {
-        s
-    } else if let Some(s) = tags.get("psv:lanes:forward") {
-        s
-    } else if oneway {
-        if let Some(s) = tags.get("bus:lanes") {
-            s
-        } else if let Some(s) = tags.get("psv:lanes") {
-            s
-        } else {
-            ""
-        }
-    } else {
-        ""
-    };
-    if !fwd_bus_spec.is_empty() {
-        let parts: Vec<&str> = fwd_bus_spec.split('|').collect();
-        let offset = if fwd_side[0].lt == LaneType::SharedLeftTurn {
-            1
-        } else {
-            0
-        };
-        if parts.len() == fwd_side.len() - offset {
-            for (idx, part) in parts.into_iter().enumerate() {
-                if part == "designated" {
-                    fwd_side[idx + offset].lt = LaneType::Bus;
-                }
-            }
-        }
-    }
-    if let Some(spec) = tags
-        .get("bus:lanes:backward")
-        .or_else(|| tags.get("psv:lanes:backward"))
-    {
-        let parts: Vec<&str> = spec.split('|').collect();
-        if parts.len() == back_side.len() {
-            for (idx, part) in parts.into_iter().enumerate() {
-                if part == "designated" {
-                    back_side[idx].lt = LaneType::Bus;
-                }
+            // Use our own widths for the moment
```

Comment:
I'd like to cutover ASAP and iterate on smaller issues. I realized the width issue is easy to workaround -- just override with the current A/B Street config. We can remove this later.